packages/core/template/default/index.md

```diff
@@ -1,132 +1,75 @@
 <frontmatter>
-  title: Landing Page
+  title: Home Page
   layout: default.md
-  pageNav: 2
+  pageNav: 4
   pageNavTitle: "Chapters of This Page"
```

Comment:
perhaps a more appropriate nav title here

Reply:
Fixed in [945ea73](https://github.com/MarkBind/markbind/pull/2225/commits/945ea738078da9000f4299903da634c4db9dee5e)!